summationOfPrimes.js

```diff
@@ -1,3 +1,25 @@
 module.exports = function(limit) {
-  // do work here
-}
+  let prime = [2];
+  let sum = 0;
+
+  for (let i = 3; i < limit; i += 2) {
+    let isPrime = true;
+
+    for (let j = 0; j < prime.length && prime[j] * prime[j] <= i; j++) {
+      if (i % prime[j] === 0) {
+        isPrime = false;
+        break;
+      }
+    }
+
+    if (isPrime) {
+      prime.push(i);
+    }
+  }
+
+  for (let k = 0; k < prime.length; k++) {
```

Comment:
I'm approving this, but think of a way to do the summation within the two for loops as an optimization.